packages/eslint-plugin-template/src/rules/conditional-complexity.ts

```diff
@@ -21,37 +25,41 @@ export default createESLintRule<Options, MessageIds>({
   meta: {
     type: 'suggestion',
     docs: {
-      description: `The condition complexity shouldn't exceed a rational limit in a template.`,
+      description:
+        'The conditional complexity should not exceed a rational limit',
       category: 'Best Practices',
       recommended: false,
     },
     schema: [
       {
         type: 'object',
-        properties: {
-          maxComplexity: {
-            type: 'number',
-            minimum: 1,
-          },
-        },
+        properties: { maxComplexity: { minimum: 1, type: 'number' } },
         additionalProperties: false,
       },
     ],
     messages: {
       conditionalСomplexity:
-        'The condition complexity "{{totalComplexity}}" exceeds the defined limit "{{maxComplexity}}"',
+        'The conditional complexity "{{totalComplexity}}" exceeds the defined limit "{{maxComplexity}}"',
     },
   },
   defaultOptions: [{ maxComplexity: 5 }],
   create(context, [{ maxComplexity }]) {
+    ensureTemplateParser(context);
```

Comment:
I had to change the signature of `ensureTemplateParser` in order to compile:

![Captura de Tela 2021-01-04 às 02 20 24](https://user-images.githubusercontent.com/11965907/103503571-6f261f00-4e33-11eb-81a2-441f2911113c.png)